sale_blanket_order/views/sale_blanket_order_views.xml

```diff
@@ -181,10 +176,8 @@
                                             'quantity': original_uom_qty,
                                             'company_id': parent.company_id
                                         }"
-                                        attrs="{
-                                            'required': [('display_type', '=', False)],
-                                            'invisible': [('display_type', '=', True)],
-                                        }"
+                                        required="display_type == False"
+                                        invisible="display_type == True"
```

Comment:
Please use `column_invisible` see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0